packaging/build_all_debs

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Builds the libsxg deb packages for multiple debian family OS with Docker.
+# This script must be run at the root directory of the libsxg source tree.
+# Generated deb packages will put in the directory with the same name of
+# Dockerfiles.
+
+set -ex
+readonly DEB_DOCKER="packaging/dockerfiles/deb.dockerfile"
+declare -ar DISTROS=(
```

Comment:
Just using `readonly` might be better in favor of consistency:

```bash
readonly DISTROS=(
  ...
)
```

(I personally write everything with `declare` instead, but it's a matter of taste and I'm fine with `readonly`.)

Reply:
done.